src/EventService.Models.Dto/Models/CategoryInfo.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using LT.DigitalOffice.EventService.Models.Dto.Enums;
+
+namespace LT.DigitalOffice.EventService.Models.Dto.Models;
+
+public class CategoryInfo
+{
+  public Guid Id { get; set; }
+  public string Name { get; set; }
+  public CategoryColor Color { get; set; }
```

Comment:
Can category not have color?

Reply:
Honestly, not sure. I think the category can be without color. You suggest to use "CategoryColor?"?

Reply:
+